here-naksha-app-service/src/test/resources/unit_test_data/TC0500_updateFeatures/response_no_geometry.json

```diff
@@ -0,0 +1,19 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "id": "my-custom-id-301-1",
+      "type": "Feature",
+      "properties": {
+        "speedLimit": "50"
+      }
+    },
+    {
+      "id": "my-custom-id-301-2",
+      "type": "Feature",
+      "properties": {
+        "speedLimit": "70"
+      }
+    }
+  ]
+}
```

Comment:
the current design is that all PUT API will only modify features already existing, otherwise an error will be returned, so no upsert involved yet (so "inserted" will not be applicable for this API). Is it still the case?

I add the enum to specify whether "inserted" or "updated" should be used in the latest commit.

Reply:
please ignore that question, I see now the next issue and in the chat :D